src/modules/LanItems.rb

```diff
@@ -1088,6 +1081,45 @@ def Read
       nil
     end
 
+    # Clears internal cache of the module to default values
+    #
+    # TODO: LanItems consists of several sets of internal variables.
+    # 1) cache of items describing network interface
+    # 2) variables used as a kind of iterator in the cache
+    # 3) variables which keeps (some of) attributes of the current item (= item
+    # which is being pointed by the iterator)
+    def reset_cache
+      LanItems.Items = {}
+
+      @modified = false
+    end
+
+    # Imports data from AY profile
+    #
+    # As network related configuration is spread over the whole AY profile
+    # the function requires hash map with whole AY profile.
```

Comment:
Not quite. It is not the whole profile but rather its `<profile>/<networking>` element, AFTER it has been processed by [FromAY](https://github.com/yast/yast-network/blob/6ca4d37d55fd1517751cef86b696e9692dedbd53/src/clients/lan_auto.rb#L169).